config/database.yml

```diff
@@ -17,3 +17,5 @@ production:
   username: <%= ENV['POSTGRES_USER'] %>
   password: <%= ENV['POSTGRES_PASSWORD'] %>
   host:     <%= ENV['POSTGRES_HOST'] %>
+  sslmode: verify-full
+  sslrootcert: root.crt
```

Comment:
I think we need to provide the full path to the bundle, otherwise it fails to find it:
```Caused by:
PG::ConnectionBad: root certificate file "root.crt" does not exist (PG::ConnectionBad)
Either provide the file or change sslmode to disable server certificate verification.```

What I usually did in the past is expose the path to the bundle in the docker image and then use that ENV variable here.